app/views/user/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="user-post-container">
+  <div class="user-container">
+        <%= image_tag @user.photo, class: "profile-pic", alt: "User Profile Photo" %>
+         <div class="post-details">
+           <h1><%= @user.name %></h1>
+           <p class="total-posts">Number of posts: <%= @user.posts_counter %></p>
+         </div>
+  </div>
+  <div class="bio-container"
+      <span>Bio</span>
+      <p class="bio"><%= @user.bio %></p>
+  </div>
+
+  <% if @user.three_most_recent_posts == [] %> 
+  <div class="post_container">There are no posts by this user.</div>
+<% else %>
+<div class="post_container">
+  <% @user.three_most_recent_posts.empty.each_with_index do |post, index| %>
```

Comment:
- [ ] It will be great to ensure that the app doesn't break. It seems there is a problem in this line. Since, in the error below, the empty method is not found. This is when i click on a user and try to navigate to the user show page. kindly fix this issue ⬇️ 
![Screenshot 2023-11-02 at 10 30 03](https://github.com/noelfoka/Blog_app/assets/32361963/8eadca19-bb8d-474f-874e-25b838e9594a)

Reply:
It is done